documentation/src/theme/Navbar/index.tsx

```diff
@@ -1,225 +0,0 @@
-import React, {useCallback, useState, useEffect} from 'react';
```

Comment:
Are these file auto deleted when build with new `docusarus` version

Reply:
No, I have deleted them manually. They seem to be made just for getting around arbitrary limitations in the Docusaurus 2 alpha which we no longer need to worry about. But I'm going to do a closer check today, to see what exactly we needed them for.